modules/rosa-basic-concepts.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// understanding-rosa/understanding-rosa.adoc
+
+
+[id="rosa-basic-concepts"]
+= Basic Concepts
+
+This topic explains some basic concepts about the container platform.
+
+[id="rosa-kubernetes_{context}"]
+== Kubernetes
+
+{product-title} (ROSA) uses Red Hat’s enterprise Kubernetes platform. Kubernetes is an open source platform for managing containerized workloads and services across multiple hosts, and offers management tools for deploying, automating, monitoring, and scaling containerized apps with minimal to no manual intervention. For complete information about Kubernetes, see the link:https://kubernetes.io/docs/home/?path=users&persona=app-developer&level=foundational[Kubernetes documentation].
```

Comment:
From IBMSG, presumably this extends to the Red Hat company name as well:

"Do not use ’s to show the possessive form of IBM unless you refer to only the company
and do not identify goods or services."

Not sure how to best rewrite here... maybe "ROSA uses the enterprise Kubernetes platform from Red Hat."

Reply:
Done.